apis/gcp_config.py

```diff
@@ -16,18 +16,23 @@
 
 import dataclasses
 from apis import metric_config
+from typing import Optional
 
 
 @dataclasses.dataclass
 class GCPConfig:
   """This is a class to set up configs of GCP.
 
   Attributes:
-    project_name: The name of a project to run a test job.
+    project_name: The name of a project for Composer env to run a test job.
     zone: The zone to run a test job.
-    dataset_name: The option of dataset for metrics
+    dataset_name: The option of dataset for metrics.
+    cluster_project: The project of a cluster, i.e. a cluster running xpk workloads.
+    cluster_name: The name of a cluster that has provisioned resources.
   """
 
   project_name: str
   zone: str
   dataset_name: metric_config.DatasetOption
+  cluster_project: Optional[str] = None
```

Comment:
I wonder if it makes sense to specialize `GCPConfig` for GKE tests

Reply:
This makes sense to me. We may have to revisit cases where we use the `composer` project to support running a local airflow orchestrator for testing.